src/olympia/scanners/tasks.py

```diff
@@ -272,12 +278,15 @@ def run_yara_query_rule(query_rule_pk):
     # Build the workflow using a group of tasks dealing with 250 files at a
     # time, chained to a task that marks the query as completed.
     chunk_size = 250
+    chunked_tasks = create_chunked_tasks_signatures(
+        run_yara_query_rule_on_versions_chunk, list(pks), chunk_size,
+        task_args=(query_rule_pk,))
     workflow = (
-        create_chunked_tasks_signatures(
-            run_yara_query_rule_on_versions_chunk, list(pks), chunk_size,
-            task_args=(query_rule_pk,)) |
+        chunked_tasks |
```

Comment:
I find this notation quite confusing actually (the `|`) but that was there before so 🤷‍♂